Entities/Golem/EnemyGolemAI.cs

```diff
@@ -0,0 +1,64 @@
+using Godot;
+using Godot.Collections;
+using System;
+using System.Linq;
+using System.Reflection.Metadata.Ecma335;
+
+namespace ApproachTheForge {
+
+	public partial class EnemyGolemAI : GolemAI
+	{
+		protected override Bearing ObjectiveBearing => Bearing.Left;
+
+		public override void _PhysicsProcess(double delta)
+		{
+			this._Velocity = Velocity;
+
+			// Add the gravity.
+			if (!IsOnFloor())
+			{
+				this._Velocity.Y += gravity * (float)delta;
+			}
+
+			if(SearchForPlayerEntity())
+			{
+				this.FaceTarget();
+			}
+			else
+			{
+				this.FaceObjective();
+			}
+
+			if(this.FindWalls())
+			{
+				this.Jump();
+			}
+
+			this._Velocity.X = (int)this.Bearing * Speed;
+
+			this.Velocity = this._Velocity;
+
+			MoveAndSlide();
+		}
+
+		private bool SearchForPlayerEntity()
+		{
```

Comment:
Move this call to `_Ready()`

Reply:
It looks like it didn't correctly highlight what I mean to comment on. I was referening the `GetNode` call in the method.

Reply:
Diiid it